pages/Add/index.js

```diff
@@ -1,7 +1,131 @@
+import styled from "styled-components";
+import { useEffect, useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import {
+  setLocalStorage,
+  loadLocalStorage,
+} from "../../components/LocalStorage";
+import { nanoid } from "nanoid";
+
 export default function Add() {
+  const [startDate, setStartDate] = useState(new Date());
+  const [fishList, setFishList] = useState(
+    loadLocalStorage("localfFishList") ?? []
+  );
+
+  useEffect(() => {
+    setLocalStorage("localfFishList", fishList);
+  });
```

Comment:
I think you have to add empty square-brackets to line 19, so that this useEffect takes places only once the app.js is mounted

```suggestion
  },[]);
```

Reply:
An empty dependency array updates your localStorage once and then never again. But if you want to update it whenever your fishList updates, you should put fishList inside the array :)